scripts/make-localhost-token-from-dev-token.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+DEV_TOKEN=$1
+if [ "$DEV_TOKEN" == "" ]
+then
+  echo "Missing required argument: dev token"
+  echo "Usage: $0 <dev token>"
+  exit 1
+fi
+
+echo "{\"endpoint\":\"localhost\",\"api_key\":\"${DEV_TOKEN}\"}" |base64
```

Comment:
I don't think we need this script anymore?

Reply:
we definitely need it, the raw token you get from the server's console output is not in a format that the SDK knows what to do with. it expects it to be wrapped in base64 JSON. That's what the console does and that's what users supply to the SDK.

Reply:
I have this code in my side branch and it seems to work just fine?
```
const credsProvider = new StringMomentoTokenProvider({
  authToken:
    'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJsYXB0b3BAbW9tZW50b2hxLmNvbSIsImVtYWlsIjpudWxsLCJpc3MiOm51bGwsInZlciI6bnVsbCwiY3AiOiJzbyBpdCBsb29rcyBsaWtlIGEgbGVnYWN5IHRva2VuIiwiYXVkIjpudWxsfQ.gIMozwpfsh8JpCvLQwgfYhv2Qs7k8VnZrsKL233Ojs8sStfiCU71zxR290n2G9SuIF50tfLc0MR9e03HGCfM_A',
  cacheEndpoint: '0.0.0.0:9002',
  controlEndpoint: '0.0.0.0:9002',
 });
 ```

Reply:
okay, yeah, that works as a matter of happenstance since you override the endpoints, but it is definitely not a token format that the SDKs are intended to support.